SPECS-EXTENDED/GeoIP/GeoIP.spec

```diff
@@ -15,7 +15,6 @@ BuildRequires:	gcc
 BuildRequires:	make
 BuildRequires:	sed
 BuildRequires:	zlib-devel
-Requires:	GeoIP-data
```

Comment:
I was wondering if we can use `GeoIp-Lite` in here. 
As per my initial research, it seems the above package supersedes this package as per [this](https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS-EXTENDED/GeoIP-GeoLite-data/GeoIP-GeoLite-data.spec#L23)

Reply:
I can do. My opinion was for it to not have any hard dependencies and let users feed in a data source

Reply:
So, I don't think this will work. Looking at the Mariner upstream [repos](https://packages.microsoft.com/cbl-mariner/2.0/prod/extended/x86_64/Packages/g/), i don't see GeoIp-Lite. I do see Geolite-[asn|city|country]. Which is provided by the [spec file ](https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS-EXTENDED/geolite2/geolite2.spec). Do you want me to add

```
Requires GeoIP-data | GeoIp-Lite | Geolite-[asn|city|country]
```

Instead?

Reply:
If the library is functional with a user-provided data source, then I think we're the clear to remove the requirement on a data package and consider adding a `Recommends` link to `Geolite` data at a later date. Let's defer that decision to unblock @AZaugg.

Reply:
Apologies for the late response - I was out on vacation. I concur with @oliviacrain.
Thanks OC